frontend/src/components/navigation/Navigation.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { FooterWrapper } from "./NavigationStyled";
+import Bookmark from "../../assets/bookmark.svg";
+import Cart from "../../assets/cart.svg";
+import Home from "../../assets/home.svg";
+import Menu from "../../assets/menu.svg";
+import Search from "../../assets/search.svg";
```

Comment:
You could build a Icon.js where you import all icons and can easy export them
`import {Bookmark, Cart, Home, Menu, Serach} from "../../Icon"`

Reply:
is there a typical folder to put this .js ?

Reply:
I'd just call the folder "images" or even "icons" :)